spec/index.html

```diff
@@ -291,8 +291,10 @@ <h3>Object Lists</h3>
     <h3>IRIs</h3>
 
     <p>
-      <a data-cite="RDF12-CONCEPTS#dfn-iri">IRIs</a> may be written as relative or absolute IRIs or prefixed names.
-      Relative and absolute IRIs are preceded by <code>&lt;</code> (code point <span class="codepoint">U+003C</span>),
+      <a data-cite="RDF12-CONCEPTS#dfn-iri">IRIs</a> may be written as <a data-cite="RFC3986#section-5">resolved</a> IRIs,
+      <a data-cite="RDF12-CONCEPTS#dfn-relative-iri">Relative IRI references</a>,
+      or <a>prefixed names</a>.
+      Relative and resolved IRIs are preceded by <code>&lt;</code> (code point <span class="codepoint">U+003C</span>),
```

Comment:
That's an "URI-reference" in RFC 3986 speak. 
But using the RDF 1.2 names for consistency across documents is fine.